spec/lib/tasks/import_spec.rb

```diff
@@ -52,4 +52,38 @@
       )
     end
   end
+
+  describe "import:whitehall_documents" do
+    before do
+      allow($stdout).to receive(:puts)
+      Rake::Task["import:whitehall_documents"].reenable
+    end
+
+    it "creates pending whitehall migration document imports" do
+      allow(WhitehallDocumentImportJob).to receive(:perform_later)
+
+      expect {
+        Rake::Task["import:whitehall_documents"].invoke("123 456")
+      }.to change {
+        WhitehallMigration::DocumentImport.pending.exists?(whitehall_document_id: 123)
```

Comment:
Didn't know you could query like this, looks good! 👍

Reply:
Me neither, found it by accident - cool 'eh!